mico-core/src/main/java/io/github/ust/mico/core/dto/MicoServiceInterfaceDTO.java

```diff
@@ -20,14 +20,19 @@
 package io.github.ust.mico.core.dto;
 
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
-
+import io.github.ust.mico.core.configuration.extension.CustomOpenApiExtentionsPlugin;
+import io.github.ust.mico.core.model.MicoService;
+import io.github.ust.mico.core.model.MicoServiceInterface;
+import io.swagger.annotations.ApiModelProperty;
+import io.swagger.annotations.Extension;
+import io.swagger.annotations.ExtensionProperty;
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
 import lombok.experimental.Accessors;
 
 /**
- * Represents a service in Kubernetes for a {@link io.github.ust.mico.core.model.MicoService}.
+ * Represents a service in Kubernetes for a {@link MicoService}.
```

Comment:
Suggestion: DTO for a {@link MicoServiceInterface} that is mapped to a Kubernetes Service.